tests/test_pytest_cov.py

```diff
@@ -1231,8 +1231,9 @@ def test_run_target():
     except:
         p.terminate()
         raise
-    else:
-        p.close()
+    # new in python 3.7
+    #else:
+    #    p.close()
     finally:
         p.join()
 ''')
```

Comment:
Enable/use it with `sys.version_info >= (3, 7)` then?

Reply:
Good point!